hail/python/test/hail/methods/test_skat.py

```diff
@@ -105,7 +186,124 @@ def test_linear_skat_no_weights_R_truth():
     assert result.fault == 0
 
 
-def test_linear_skat_R_truth():
```

Comment:
This test is inadvertently duplicated in main. I just removed it entirely, but the diff is a bit awkward.a